app/controllers/items_controller.rb

```diff
@@ -7,4 +7,17 @@ def index
   def show 
     @item = Item.find(params[:id])
   end
+
+  def update
+    @item = Item.find(params[:id])   
+    @merchant = @item.merchant
+    if params[:disable]
+      @item.update(status: params[:disable].to_i)
+      redirect_to merchant_items_path(@merchant.id)
+    end 
```

Comment:
Should this be an elsif or else rather than two separate if statements?

Reply:
I was thinking since the user has the opportunity to do both enable and disable at the same time, it makes sense to have two separate if statements because one doesn't rely on the other.

Reply:
hmmmmm, I think I agree with Sierra. In this case I think it would make more sense to put them in the same if-statement. Mainly because we can't pass a `disable` status and an `enable` status at the same time.

Reply:
Ahhh gotcha, made the changes!